examples/dataflow-flex-cicd/cloudbuild.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+steps:
+  - name: gcr.io/cloud-builders/git
+    id: 'clone-source-code'
+    args: ['clone', '$_REPO_CLONE_URL']
+  - name: gcr.io/google.com/cloudsdktool/cloud-sdk
+    id: 'build-dataflow-template-container'
+    entrypoint: 'gcloud'
+    dir: '$_REPO_DIR/'
+    args: [ 'builds', 'submit', '--project=$PROJECT_ID', '--tag', 'gcr.io/$PROJECT_ID/$_TARGET_GCR_IMAGE']
+  - name: 'gcr.io/cloud-builders/gcloud'
```

Comment:
write short shell script and combine w/ above step

Reply:
The build file now has two steps
 1) building container image 
2) Deploy spec image (shell script combining two actions). 
I see building the container image as a standalone step. WDYT?

Reply:
this is fine.